style.css

```diff
@@ -0,0 +1,108 @@
+@import url('https://fonts.googleapis.com/css2?family=Crete+Round:ital@0;1&family=Inter:wght@100;200;300;400;500;600;700;800;900&family=Poppins:ital,wght@0,100;0,200;0,300;0,400;0,500;0,600;0,700;0,800;0,900;1,100;1,200;1,300;1,400;1,500;1,600;1,700;1,800;1,900&family=Roboto:ital,wght@1,900&display=swap');
+
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+.main-container {
+  color: whitesmoke;
+  background-color: #1c1a19;
+  min-width: 375px;
+  width: 100%;
+}
+
+.logo {
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  background: #3e3c3c;
+  padding-left: 24px;
+  padding-right: 24px;
+  height: 92px;
+  align-items: center;
+}
+
+.me {
+  font-family: 'Poppins', sans-serif;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 20px;
+  line-height: 20px;
+  display: flex;
+  align-items: center;
+  text-align: center;
+  color: #fff;
+  height: 22px;
+  width: 80px;
+  left: 142px;
+}
+
+.initial {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  flex-wrap: wrap;
+  min-height: 592px;
+  height: 100vh;
+  min-width: 375px;
+  width: 100%;
+  padding-left: 24px;
+  padding-right: 24px;
+  background: url("images/illustration-header-mobile.png");
+  background-repeat: no-repeat;
+  background-position: right bottom;
+}
+
+.information {
+  font-family: 'Poppins', serif;
+  font-style: normal;
+  font-weight: 400;
+  letter-spacing: 0.4px;
+  font-size: 55px;
+  line-height: 70px;
+  color: #f55800;
+}
+
+h2 {
+  font-family: 'Poppins', sans-serif;
+  font-size: 24px;
+  font-weight: 700;
+  line-height: 32px;
+  text-align: left;
+}
+
+p {
+  font-family: 'Poppins', sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 20px;
+  line-height: 24px;
+  color: #f4f5f7;
+  margin-top: 12px;
+}
+
+ul {
+  width: 49%;
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
```

Comment:
Good job 👌but don't you think it'll be much better if you add the `align-items: center` property? This will make your design much more similar to the Figma template.